notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Maybe something like:</p><blockquote>for electrode in range(N_ELECTRODES):</blockquote><blockquote>	sign_df['c' + electrode].values</blockquote><p>could be easier to read? (so that we don't hard code range(1, 9)).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>